### ISSUE TITLE
Add wasm-bindgen feature to README/docs

### ISSUE THREAD

**Issue by qdot** [async-rs/futures-timer#64]
Just found out there's a wasm-bindgen feature via the PR then reading the Cargo.toml. Would be nice to have this in the README.

**Comment by najamelan** [comment]
Wouldn't it be better to automatically turn these dependencies on on wasm? AFAICT the library can't work on wasm without them. It would save users from having to fiddle with feature flags.

**Comment by qdot** [comment]
Sure, that'd work too.

**Comment by dariusc93** [comment]
> Wouldn't it be better to automatically turn these dependencies on on wasm? AFAICT the library can't work on wasm without them. It would save users from having to fiddle with feature flags.

I agree that it would be better to have it enabled based on the target, especially since it doesnt fail to build on non-wasm32 targets, it makes it hard to debug the `time not implemented on this platform` panic message when `Instant` or `SystemTime` is used from std due to `Instant` being used by the native module.
